code/app/src/main/java/com/example/goldencarrot/data/model/user/UserFacility.java

```diff
@@ -0,0 +1,47 @@
+package com.example.goldencarrot.data.model.user;
+
+public class UserFacility {
```

Comment:
Why do we need this class? This is very similar to the User class that we already have. We should add on top of that class to inherit most of this methods

A regular user already has location, imageUrl, email, name